src/discrete-layer/components/job-manager/jobs.dialog.tsx

```diff
@@ -319,12 +319,14 @@ export const JobsDialog: React.FC<JobsDialogProps> = observer((props: JobsDialog
         </DialogTitle>
         <DialogContent className="jobsBody">
           {renderDateTimeRangePicker()}
-          {renderGridList()}
+          {!error && renderGridList()}
           
           {
-            mutationQuery.error !== undefined && (
+            (mutationQuery.error !== undefined || error) && (
               // eslint-disable-next-line
-              <GraphQLError error={mutationQuery.error} />
+              <div className={`${error && 'render-jobs-data-errror'}`}>
```

Comment:
Preferable use **Box** rather than **div** [if needed]
consider in this case use syntax as following 
                className={`${error ? 'render-jobs-data-errror' : ''}`}